README.md

```diff
@@ -134,6 +134,13 @@ Optionally, you can specify a region in which to store the user's data, if you w
   let trusona = Trusona(region: .asiaPacific)
 ```
 
+3. Create a `Trusona` object in the `ViewController` class using a custom domain name. If you have deployed your own infrastructure, you can customize the SDK to use it by providing the base url that points to it, as shown below:
```

Comment:
Two observations:

- Maybe instead of defining this as item #3 we could have it as a separate section? Currently we have:

1. Import `TrusonaSDK` module
2. Create a `Trusona` object in the `ViewController` class.
3. Create a `Trusona` object in the `ViewController` class using a custom domain name.

So it would appear as if you need to execute all three steps.

- Also, can we have a constructor that only takes the `endpointConfig` field?  When called it would default to a nil logger and urlSession.

Reply:
agreed

Reply:
ok, fixing it

Reply:
fixed